kubernetes-model-generator/kubernetes-model-core/src/main/java/io/fabric8/kubernetes/api/model/KubernetesList.java

```diff
@@ -71,8 +71,6 @@ public String getApiVersion() {
 
     @Override
     public List<HasMetadata> getItems() {
-        List<HasMetadata> sortedItems = new ArrayList<>(super.getItems());
-        Collections.sort(sortedItems, new HasMetadataComparator());
-        return sortedItems;
+      return super.getItems();
     }
 }
```

Comment:
Why don't we just remove the method?

Reply:
Because of the other overrides: https://github.com/fabric8io/kubernetes-client/blob/9025af6260443ddb4d6f5613f66eb0ad6c8c0905/kubernetes-model-generator/kubernetes-model-core/src/main/java/io/fabric8/kubernetes/api/model/KubernetesList.java#L61 - I can't imagine that's still relevant though.  I'm also going to remove these classes eventually #3906